nginx/containerbuddy.json

```diff
@@ -0,0 +1 @@
+{"consul":"${CONSUL}:8500","onStart":"/opt/containerbuddy/reload.sh preStart","services":[{"name":"nginx","port": 80,"publicIp": true,"health":"/usr/bin/curl --fail -s http://localhost/health.txt","poll":10,"ttl":25}],"backends":[{"name":"wordpress","poll":5,"onChange":"/opt/containerbuddy/reload.sh"}]}
```

Comment:
Ack, my mistake. it's still the `onStart` event, but the `preStart` function in there. Apologies.